### PR TITLE
 Relax the the constraints for the average CPU usage during performance scheduler tests.

### LINKED ISSUE #2240: scalability: "Average CPU usage" flakes
/kind flake
**What happened**:
The scalability test failed on the "Average CPU usage 504mCpu is greater than maximum expected 500mCPU".

```shell
{Failed  === RUN   TestScalability/CommandStats
    checker_test.go:87: Average CPU usage 504mCpu is greater than maximum expected 500mCPU
--- FAIL: TestScalability/CommandStats (0.00s)
}
```

**What you expected to happen**:
No error happened.

**How to reproduce it (as minimally and precisely as possible)**:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/2235/pull-kueue-test-scheduling-perf-main/1792493875051368448

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Kueue version (use `git describe --tags --dirty --always`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by trasc**
#### What type of PR is this?

/kind bug
/kind flake

#### What this PR does / why we need it:
Relax the the constraints for the average CPU usage during performance scheduler tests.
#### Which issue(s) this PR fixes:

Fixes #2240 

#### Special notes for your reviewer:

There is no indication of a systematic usage growth analyzing the periodic runs making the failed test just a "victim"  of CI environment status , in my opinion. However let's extend the accepted range wit 5%.  

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: test/performance/scheduler/default_rangespec.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ cmd:
   # Average value 351116.4 (+/- 0.9%), setting at +20%
   maxWallMs: 425_000
 
-  # Average value 396 mCPU (+/- 8%), setting at +25%
-  mCPU: 500
+  # Average value 396 mCPU (+/- 8%), setting at +30%
```
Comment (tenzen-y) [inline test/performance/scheduler/default_rangespec.yaml:11]: I'm wondering if we should re-measure the average value before we release v0.7.0.
@mimowo @alculquicondor WDYT?

**Review by tenzen-y — COMMENTED** [review]
/approve

**Comment by tenzen-y** [comment]
/retest 
due to https://github.com/kubernetes-sigs/kueue/issues/2243

File: test/performance/scheduler/default_rangespec.yaml (hunk 1 above)
Comment (mimowo) [reply test/performance/scheduler/default_rangespec.yaml:11]: Measuring this currently by hand is fairly time-consuming, so at some point I would like to do the script: https://github.com/kubernetes-sigs/kueue/issues/2077. 

Until then, I think we are ok with the current approach. If the thresholds remain too strong we can relax them by cherry-pick.

**Comment by mimowo** [comment]
Thanks
/lgtm

File: test/performance/scheduler/default_rangespec.yaml (hunk 1 above)
Comment (tenzen-y) [reply test/performance/scheduler/default_rangespec.yaml:11]: That makes sense. After we implemented the script, I would recommend re-running every minor release (v0.8, v0.9...)

**Pull request merged**